annotation/src/main/java/org/cbioportal/genome_nexus/annotation/service/AnnotationEnricher.java

```diff
@@ -1,11 +0,0 @@
-package org.cbioportal.genome_nexus.annotation.service;
-
-import org.cbioportal.genome_nexus.annotation.domain.VariantAnnotation;
-
-/**
- * @author Selcuk Onur Sumer
- */
-public interface AnnotationEnricher
```

Comment:
This file is not removed, just renamed, for some reason github shows it removed...